astropy/io/fits/hdu/base.py

```diff
@@ -1625,7 +1625,7 @@ def _verify(self, option="warn"):
         return errs
 
 
-@deprecated("v6.0")
+@deprecated("6.0")
```

Comment:
orthogonal, but doesn't hurt

Reply:
It was changed due to my adding the version in the message which needs to be able to parse these tags. 

Since I will remove that change, I will be undoing this change as well.